shuffleAnArray.py

```diff
@@ -0,0 +1,31 @@
+class Solution(object):
+
+    def __init__(self, nums):
+        """
+        :type nums: List[int]
+        """
+        self.nums = nums
+
+    def reset(self):
+        """
+        Resets the array to its original configuration and return it.
+        :rtype: List[int]
+        """
+        result = self.nums[:]
+        return result
+
+    def shuffle(self):
+        """
+        Returns a random shuffling of the array.
+        :rtype: List[int]
+        """
+        result = self.reset()
+        length = len(result)
+        import random
```

Comment:
Also, if you need to import a module, do it at the top of the file.
We strongly discourage local imports.
Sometimes, for circular dependency reasons, you have to do local imports.
Do it at the top of a certain function.